Makefile

```diff
@@ -13,7 +13,8 @@ all: manager
 
 # Run tests
 test: generate fmt vet manifests
-	go test -v ./backend... ./errors/... ./controllers/... -coverprofile cover.out
+	docker-compose run secrets-manager-local \
```

Comment:
I'd create a docker-compose task named `test` or similar that encapsulates this whole command. So you just `docker-compose run test` for testing everything

Reply:
i agree, rolling this back...